core/src/main/scala/org/ensime/core/AnalyzerManager.scala

```diff
@@ -275,7 +266,7 @@ class AnalyzerManager(
         }
         def receive: Receive = {
           case FullTypeCheckCompleteEvent =>
-            analyzer ! RemoveFilesReq(filesToBeDeleted.toList.map(f => richEnsimeFile(f.file).toPath.toFile))
+            analyzer ! RemoveFilesReq(filesToBeDeleted.map(toFile))
```

Comment:
First I get the files whose symbols I must remove using `removeSymbolsOf` on all modules, and I pass a `TypeCheckFilesReq(_)` to the project-wide analyzer to type-check these files.
Then, when I get a `FullTypeCheckCompleteEvent`, I remove the symbols unsing `RemoveFilesReq`.
Now I forward the request (the one not specific to any module that I received) to this analyzer.